code/modules/admin/admin.dm

```diff
@@ -853,7 +853,10 @@ var/global/respawntime = 15
 	set category = "Server"
 	set desc="Sets the respawn time"
 	set name="Set Respawn Timer"
-	respawntime = time
+	if (time > 0)
```

Comment:
Larger or equal. If for some reason they want players not have to wait at all, that should be a possibility.